components/PanelErrorBanner/PanelErrorBanner.tsx

```diff
@@ -3,7 +3,7 @@ import Warning from "public/assets/icons/warning.svg";
 import styled from "styled-components";
 
 export function PanelErrorBanner() {
-  const { errorMessages } = useErrorContext();
+  const { errorMessages } = useErrorContext("panel");
```

Comment:
this is how we access different contexts, if you dont provide a label, it will just use the default to not break compatibility with previous code